apps/condo/domains/user/schema/SignInUserService.js

```diff
@@ -0,0 +1,221 @@
+        messageForUser: 'api.user.signInUser.REQUIRED_USER_DATA_IS_MISSING',
+    }),
+    NOT_SUPPORTED_USER_TYPE: {
+        mutation: 'signInUser',
+        variable: ['data', 'userType'],
+        code: BAD_USER_INPUT,
+        type: 'NOT_SUPPORTED_USER_TYPE',
+        message: `Only the following user types are supported: ${SUPPORTED_USER_TYPES.join(', ')}`,
+    },
+    DIFFERENT_PHONE_NUMBERS: {
+        mutation: 'signInUser',
+        variable: ['data', 'userData', 'phone'],
+        code: BAD_USER_INPUT,
+        type: 'DIFFERENT_PHONE_NUMBERS',
+        message: 'The verified phone number and the phone number from the payload cannot be different',
+    },
+    INVALID_PHONE_NUMBER: {
+        mutation: 'signInUser',
+        code: BAD_USER_INPUT,
+        type: 'INVALID_PHONE_NUMBER',
+        message: 'Invalid phone number',
+        messageForUser: 'api.user.signInUser.INVALID_PHONE_NUMBER',
+    },
+    USER_NOT_FOUND: {
+        mutation: 'signInUser',
+        code: NOT_FOUND,
+        type: 'USER_NOT_FOUND',
+        message: 'User not found',
+        messageForUser: 'api.user.signInUser.USER_NOT_FOUND',
+    },
+    TOKEN_NOT_FOUND: {
+        mutation: 'signInUser',
+        variable: ['data', 'token'],
+        code: NOT_FOUND,
+        type: 'TOKEN_NOT_FOUND',
+        message: 'Token not found',
+    },
+    DV_VERSION_MISMATCH: {
+        ...COMMON_ERRORS.DV_VERSION_MISMATCH,
+        mutation: 'signInUser',
+    },
+    WRONG_SENDER_FORMAT: {
+        ...COMMON_ERRORS.WRONG_SENDER_FORMAT,
+        mutation: 'signInUser',
+    },
+    DAILY_REQUEST_LIMIT_FOR_IP_REACHED: {
+        ...USER_ERRORS.DAILY_REQUEST_LIMIT_FOR_IP_REACHED,
+    },
+}
+
+const getUserPayload = (user, userData) => {
+    const payload = {}
+    for (const [key, value] of Object.entries(userData))  {
+        if (!value) continue
+        if (get(user, key)) continue
+        payload[key] = value
+    }
+    return payload
+}
+
+/**
+ * Checks that all required fields are present in a user data.
+ * @param {string[]} requiredFields
+ * @param {*} userData
+ * @return {{missingFields: string[], missingRequiredFields: boolean}}
+ */
+const checkRequiredUserFields = (requiredFields, userData) => {
+    if (!Array.isArray(requiredFields)) throw new Error('"requiredFields" should be an array')
+
+    const missingFields = []
+    for (const field of requiredFields) {
+        const value = get(userData, field, null)
+        if (isNil(value)) missingFields.push(field)
+        if (typeof value === 'string' && value.length < 1) missingFields.push(field)
+    }
+    return { missingRequiredFields: missingFields.length > 0, missingFields }
+}
+
+const SignInUserService = new GQLCustomSchema('SignInUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input SignInUserUserDataInput { phone: String, name: String, email: String, password: String, meta: JSON }',
```

Comment:
Just for the record, what are the final names of authentication mutation and queries?